docs/sources/tempo/configuration/grafana-alloy/tail-sampling/enable-tail-sampling.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Enable tail-based sampling
+menuTItle: Enable tail-based sampling
+description: Use tail-based sampling to optimize sampling decisions.
```

Comment:
We can also add more description here. I'll see if I can convert this into a Vale rule.

Reply:
```suggestion
description: Configure tail sampling with Tempo and Grafana Alloy to optimize sampling decisions.
```